src/main/common/axis.h

```diff
@@ -27,6 +27,7 @@ typedef enum {
 } axis_e;
 
 #define XYZ_AXIS_COUNT 3
+#define XY_AXIS_COUNT 2
```

Comment:
Where is this to be used?

Reply:
It is possible that this is leftover. This PR is only draft to show ideas, I didn't spend much time polishing it.